commands/start.go

```diff
@@ -0,0 +1,64 @@
+/*
+Package commands contains github-audit shell commands related logic.
+*/
+package commands
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	// github-audit pid file
+	GithubAuditPIDFile = "github-audit.process"
```

Comment:
Many places we are prefixing GithuAudit in variable names, I think we can remove in some places. e.g. instead of GithubAuditPIDFile, it can be pidFile, or if shared var then PidFile

Reply:
done